src/d3a/models/market/market_structures.py

```diff
@@ -67,23 +69,31 @@ def _to_csv(self):
         return self.id, rate, self.energy, self.price, self.seller
 
 
+def copy_offer(offer):
```

Comment:
Can you please tell the reason for this? Dont you trust deepcopy any more ;-) ?

Reply:
Explained offline, deppcopy is slow.

Reply:
As discussed offline, its performance was far far slower than creating manually the same object.